transformers/component_tests/triggers/bid_event_trigger_test.go

```diff
@@ -0,0 +1,493 @@
+			flipKickModel.ColumnValues[event.AddressFK] = flipAddressID
+			flipKickModel.ColumnValues[event.LogFK] = logID
+		})
+
+		It("gets the relevant ilk for event", func() {
+			expectedEvent := expectedBidEvent(flipKickModel, "kick", flipAddress, headerOne.BlockNumber)
+			expectedEvent.IlkIdentifier = test_helpers.FakeIlk.Identifier
+
+			flipIlkErr := flipRepo.Create(diffID, headerOne.Id, flip.IlkMetadata, test_helpers.FakeIlk.Hex)
+			Expect(flipIlkErr).NotTo(HaveOccurred())
+
+			insertErr := event.PersistModels([]event.InsertionModel{flipKickModel}, db)
+			Expect(insertErr).NotTo(HaveOccurred())
+
+			var bidEvents []bidEvent
+			queryErr := db.Select(&bidEvents, `SELECT bid_id, contract_address, act, lot, bid_amount, ilk_identifier, block_height FROM maker.bid_event`)
+			Expect(queryErr).NotTo(HaveOccurred())
+			Expect(bidEvents).To(ConsistOf(expectedEvent))
+		})
+
+		It("gets the relevant urn for event", func() {
+			expectedEvent := expectedBidEvent(flipKickModel, "kick", flipAddress, headerOne.BlockNumber)
+			expectedEvent.UrnIdentifier = common.HexToAddress("0x" + test_data.RandomString(40)).Hex()
+
+			bidUsrMetadata := types.GetValueMetadata(storage.BidUsr,
+				map[types.Key]string{constants.BidId: expectedEvent.BidID}, types.Address)
+			usrErr := flipRepo.Create(diffID, headerOne.Id, bidUsrMetadata, expectedEvent.UrnIdentifier)
+			Expect(usrErr).NotTo(HaveOccurred())
+
+			insertErr := event.PersistModels([]event.InsertionModel{flipKickModel}, db)
+			Expect(insertErr).NotTo(HaveOccurred())
+
+			var bidEvents []bidEvent
+			queryErr := db.Select(&bidEvents, `SELECT bid_id, contract_address, act, lot, bid_amount, urn_identifier, block_height FROM maker.bid_event`)
+			Expect(queryErr).NotTo(HaveOccurred())
+			Expect(bidEvents).To(ConsistOf(expectedEvent))
+		})
+	})
+
+	Describe("when diffs are inserted after events", func() {
+		var (
+			flipAddress   string
+			flipRepo      flip.FlipStorageRepository
+			diffID        int64
+			flipKickModel event.InsertionModel
+		)
+
+		BeforeEach(func() {
+			flipAddress = test_data.EthFlipAddress()
+			flipAddressID, addressErr := shared.GetOrCreateAddress(flipAddress, db)
+			Expect(addressErr).NotTo(HaveOccurred())
+			flipRepo = flip.FlipStorageRepository{ContractAddress: flipAddress}
+			flipRepo.SetDB(db)
+			diffID = CreateFakeDiffRecord(db)
+			flipKickModel = test_data.FlipKickModel()
+			flipKickModel.ColumnValues[event.HeaderFK] = headerOne.Id
+			flipKickModel.ColumnValues[event.AddressFK] = flipAddressID
+			flipKickModel.ColumnValues[event.LogFK] = logID
+			insertErr := event.PersistModels([]event.InsertionModel{flipKickModel}, db)
+			Expect(insertErr).NotTo(HaveOccurred())
+		})
+
+		Specify("inserting a flip_ilk diff triggers an update to the ilk_identifier of related bids", func() {
+			expectedEvent := expectedBidEvent(flipKickModel, "kick", flipAddress, headerOne.BlockNumber)
+			expectedEvent.IlkIdentifier = test_helpers.FakeIlk.Identifier
```

Comment:
I wonder if it would be helpful to include an assertion that the `bid_event.ilk` is null until the `flipIlk` is inserted?